### DISCUSSION TITLE
Imagem em Base64 com erro

### DISCUSSION THREAD

**Discussion by keniogta** [viniciussanchez/RESTRequest4Delphi#187] [General]
Olá Vinicius.
Tenho um server horse me enviando um array com uma imagem em 64, até ai tudo bem, no postman visualizo normalmente.

![image](https://github.com/viniciussanchez/RESTRequest4Delphi/assets/121513457/92435a1f-b6cd-41d2-bf29-60080d7bb0ab)

Agora no FMX onde busco e populo o fdMenTable e a string do 64 não bate, vem com caracteres invalidos!
A função:
![image](https://github.com/viniciussanchez/RESTRequest4Delphi/assets/121513457/671390ff-8288-4fdc-aa9f-208568c4654f)

O que vem no dataset:
![image](https://github.com/viniciussanchez/RESTRequest4Delphi/assets/121513457/83cdacbc-509d-4a24-9cde-1d2f87bd2425)

**Comment by viniciussanchez** [comment]
Bom dia... tira o adapter, pega o Resp.Content e veja como está, se assim chega certo. Se chegar, precisa analisar o que está acontecendo dentro do adapter. Provavelmente dentro do dataset-serialize para ser mais exato.

**Comment by marcelobernardi** [comment]
Alguma solução pro caso acima ? Estou com o mesmo problema.
